force-app/main/default/classes/DMLOperations.cls

```diff
@@ -48,6 +52,11 @@ public with sharing class DMLOperations {
      */
     public static void createAccount(String name, String industry) {
         // Create a new Account and set the fields inline
+        Account newAccount = new Account(Name = 'Test Account', Industry = 'Technology');
+        
+        
+        database.insert(newAccount, AccessLevel.USER_MODE);
```

Comment:
Nice to see that you are making sure that the users have permission and not running in System mode.